src/App.tsx

```diff
@@ -20,12 +62,16 @@
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={event => {
```

Comment:
The checklist advises against using one-letter variable names. Consider renaming `event` to something more descriptive, like `changeEvent`, to improve code readability.